src/main/java/picard/util/AlleleSubsettingUtils.java

```diff
@@ -0,0 +1,267 @@
+package picard.util;
+
+import htsjdk.variant.variantcontext.*;
+import picard.fingerprint.Snp;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+/**
+ * Utilities class containing methods for restricting {@link VariantContext} and {@link GenotypesContext} objects to a
+ * reduced set of alleles, as well as for choosing the best set of alleles to keep and for cleaning up annotations and
+ * genotypes after subsetting.
+ *
+ * @author David Benjamin davidben@broadinstitute.org;
+ * @author Yossi Farjoun farjoun@broadinstitute.org;
+ */
+
+public final class AlleleSubsettingUtils {
+    private AlleleSubsettingUtils() {
+    }  // prevent instantiation
```

Comment:
Why prevent instantiation? This isn't a singleton so there's no harm in someone creating an instance if they want to.

Reply:
I merely copied this from GATK...but that's not an excuse. I think that since all the members are static,  one doesn't need to instantiate in order to use. but looking at other Picard classes, such protection does not seem to be the norm in Util classes...removing.